File: PRAK204-2510817220020-LailyAzizahIslami.py
phi = 3.142857
nilai = input().split()

if len(nilai) == 1:
    r = float(nilai[0])
    t = float(input())
else:
    r, t = map(float, nilai)

volume = phi * r * r * t
luas = 2 * phi * r * (r + t)
keliling = 2 * phi * r

print(f"\nVolume = {volume:.2f}")
print(f"Luas = {luas:.2f}")
print(f"Keliling = {keliling:.2f}")

File: PRAK204-2510817220020-LailyAzizahIslami.c
#include <stdio.h>

int main(){
    float r, tinggi, volume, luas, keliling, phi = 3.142857;

    scanf ("%f %f", &r, &tinggi);
    volume = phi * (r * r) * tinggi;
    luas = 2 * phi * r * (r + tinggi);
    keliling = 2 * phi * r;
    
    printf ("\nVolume = %.2f\n", volume);
    printf ("Luas = %.2f\n", luas);
    printf ("Keliling = %.2f\n", keliling);

return 0;
}